### ISSUE TITLE
reproduce the mAP in readme

### ISSUE THREAD

**Issue by yanxurui** [eriklindernoren/PyTorch-YOLOv3#208]
How specifically can I reproduce the mAP results mentioned in readme?

I have run test.py using the weights from the original darnet but only got mAP of 0.5145.
How can I get mAP around 55?
What's the mAP using the weights trained by this implementation?

Thanks.

**Comment by CodePothunter** [comment]
Same, waiting for answer

**Comment by ShuaiZ1037** [comment]
waiting for answer

**Comment by SWHL** [comment]
I meet the same problem, waitting the answer.

**Comment by yyx1107** [comment]
@SWHL I changed the batchsize intest.py and change the cfg.file(batch=1,subdivisions=1),but it didn't work.I also get 51.45.

**Comment by yyx1107** [comment]
Can you help us?@eriklindernoren

**Comment by mvcaro** [comment]
Any pointers on this @eriklindernoren ?

**Comment by talenz** [comment]
Same mAP: 0.5145

**Comment by betterhalfwzm** [comment]
Same mAP: 0.5145

**Comment by cjnjuwhy** [comment]
same mAP: 0.5145

**Comment by BaiqiangGit** [comment]
same mAP: 0.5145

**Comment by dhkwon1122** [comment]
There is a bug in reading Detection label and larger batch makes the problem worse.
See https://github.com/eriklindernoren/PyTorch-YOLOv3/issues/243
